main/options/options.cc

```diff
@@ -409,6 +409,8 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                     cxxopts::value<string>()->default_value(empty.errorUrlBase), "url-base");
     options.add_options("advanced")("experimental-ruby3-keyword-args",
                                     "Enforce use of new (Ruby 3.0-style) keyword arguments", cxxopts::value<bool>());
+    options.add_options("advanced")("typed-super", "Enable typechecking of `super` calls when possible",
+                                    cxxopts::value<bool>()->default_value("true"));
```

Comment:
One of the nice (and maybe sometimes frustrating) things about Sorbet is that there aren't a profusion of options affecting how we do typechecking.  I didn't see an explanation of why we're adding this option here that does control that aspect.

My guess is that (as mentioned in the docs) there's no real `T.unsafe` escape hatch for `super` calls as there is for normal method calls, and so we need to provide this as a big hammer.  Is that roughly the idea?

Reply:
That’s the idea. I’m open to chat about this more